pennylane/operation.py

```diff
@@ -429,10 +429,6 @@ def __init__(self, *params, wires=None, do_queue=True):
         if do_queue:
             self.queue()
 
-    def __str__(self):
```

Comment:
Sorry, these are some convenience changes from https://github.com/PennyLaneAI/pennylane/pull/883. Hope we merge it soon...